src/SFA.DAS.ProviderCommitments/SFA.DAS.ProviderCommitments.Web/Views/Apprentice/Price.cshtml

```diff
@@ -0,0 +1,34 @@
+﻿@using SFA.DAS.CommitmentsV2.Shared.Extensions
+@using SFA.DAS.ProviderCommitments.Web.RouteValues
+@model SFA.DAS.ProviderCommitments.Web.Models.Apprentice.PriceViewModel
+
+@{
+    ViewBag.Title = "What's the new agreed apprenticeship price?";
+    ViewBag.GaData.Vpv = "/apprentices/change-employer/price";
+}
+
+<div class="govuk-grid-row">
+    <div class="govuk-grid-column-two-thirds">
+        <partial name="_ValidationSummary" />
+        <h1 class="govuk-heading-xl">@ViewBag.Title</h1>
+        <p>This is the price agreed between you and the new employer to complete the remainder of the apprenticeship training and assessment.</p>
+        <form method="post">
+            @Html.HiddenFor(x => x.EmployerAccountLegalEntityPublicHashedId)
+            @Html.HiddenFor(x => x.StartDate)
+            <div class="govuk-form-group  @Html.AddClassIfPropertyInError(x => x.Price, "govuk-form-group--error ")">
+                @Html.ValidationMessageFor(m => m.Price, null, new { @class = "govuk-error-message", id = "error-message-" + Html.IdFor(m => m.Price) })
```

Comment:
The error state does not exactly match the screenshot. The error message should be immediately next to the input, and the explanatory note should be within the red-bordered section.